awx/main/models/unified_jobs.py

```diff
@@ -1009,6 +1009,8 @@ def result_stdout_raw_handle(self, enforce_max_bytes=True):
                 dir=settings.JOBOUTPUT_ROOT,
                 encoding='utf-8'
             )
+            from awx.main.tasks import purge_old_stdout_files  # circular import
```

Comment:
The only time we actually *write* stdout files (as of Tower 3.3) is that when somebody downloads a *large* stdout blob, we write it to disk at `settings.JOBOUTPUT_ROOT`.

It's likely overkill to run this in a periodic task once per week; instead, this makes it so that cleanup happens passively every time you download large stdout.